FirebasePerformance/Sources/Configurations/FPRRemoteConfigFlags.m

```diff
@@ -110,8 +118,10 @@ - (void)update {
 
   NSTimeInterval timeIntervalSinceLastFetch =
       [self.fprRemoteConfig.lastFetchTime timeIntervalSinceNow];
-  if (!self.fprRemoteConfig.lastFetchTime ||
-      ABS(timeIntervalSinceLastFetch) > kFPRConfigFetchIntervalInSeconds) {
+  NSTimeInterval timeSinceAppStart = [kFPRAppStartTime timeIntervalSinceNow];
+  if ((ABS(timeSinceAppStart) > self.appStartConfigFetchDelayInSeconds) &&
```

Comment:
When is `update()` gets called? Will it be possible that `update()` always gets called within 30 seconds, so that the remote config will never be fetched?

Reply:
update() is called whenever we bring the app to foreground/app launch.

So, this 30 seconds is applied only for a cold start of the application. For the all subsequent launches, this delay does not kick in.

Reply:
Makes sense! Thanks!